listenbrainz/webserver/static/js/src/recommendations/RecommendationCard.tsx

```diff
@@ -234,56 +240,64 @@ export default class RecommendationCard extends React.Component<
                 aria-labelledby="recommendationControlsDropdown"
               >
                 <RecommendationControl
-                  icon={faAngry}
+                  iconHover={faAngry}
+                  iconNonHover={faAngryRegular}
                   title="I never want to hear this again!"
                   action={() =>
                     feedback === "hate"
                       ? this.deleteFeedback("Like", faThumbsUpRegular)
                       : this.submitFeedback("hate", faAngry, "hate")
                   }
-                  className={`${feedback === "hate" ? " angry" : ""}`}
+                  classNameHover="on"
```

Comment:
@MonkeyDo regular/solid icons work with hover/non-hover state like we wanted to but there is one issue I am stuck at:

If I click on let us say angry face, the next time when I click on `Like` to get the dropdown the angry face is still regular and is not solid. in other words, the icons are rendered solid only on hover and not onclick. I understand what the problem is; I am not managing the classes accordingly, but I still don't understand how can I get around this. I am posting a video on IRC for your reference.